Themes/default/css/index.css

```diff
@@ -597,9 +597,9 @@ dl.settings {
 	padding: 5px;
 }
 dl.settings dt {
-	width: 56%;
+	width: 45%;
```

Comment:
Changing this will change alot in SMF, are you sure it won't break anything else ?